k8s/client.go

```diff
@@ -27,16 +27,16 @@ type Client struct {
 	Config          *rest.Config
 }
 
+var KubeConfig string
+var ContextName string
```

Comment:
```suggestion
// K8s Config and Context data
var (
    KubeConfig string
    ContextName string
)
```

Pleasing the Linter :)

Reply:
Done :+1:

Reply:
Ahh it needs the comments, else the linter will keep warning 😅 Can check the files tab.

Reply:
Added the comments.